packages/image_picker/image_picker_android/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerDelegate.java

```diff
@@ -630,7 +630,14 @@ public boolean onActivityResult(
 
   private void handleChooseImageResult(int resultCode, Intent data) {
     if (resultCode == Activity.RESULT_OK && data != null) {
-      String path = fileUtils.getPathFromUri(activity, data.getData());
+      String path;
+      // On Android 10 devices using Android Photo Picker, the Uri from getData() could be null.
+      if (data.getData() != null) {
+        path = fileUtils.getPathFromUri(activity, data.getData());
```

Comment:
This is unnecessarily duplicating `fileUtils.getPathFromUri`; only getting the URI needs to be branched.